cinemaAppFE/src/app/home/home.component.html

```diff
@@ -0,0 +1,5 @@
+<body>
```

Comment:
This should be a DIV there should be only one BODY element in the app and that is already present in index.html by default